lib/order.rb

```diff
@@ -0,0 +1,84 @@
+require_relative 'text'
+
+class Order
+
+  attr_reader :number, :basket, :total, :history
+
+  def initialize
+    @number = 1
+    @basket = []
+    @total = 0
+    @history = []
+    @text = Text.new
+  end
+
+  def add_to_basket(dish)
+    @basket << dish
+    total_so_far
+  end
+
+# viewing basket with the total cost of basket
+  def view_basket
+    @print = ["Viewing basket for order number #{@number}:\n"]
```

Comment:
This method looks a bit too lengthy. Maybe it can be refactored or split into 2?

Reply:
Agreed! I wasn't sure i liked how lengthy this was. Either of your suggestions are really useful here